packages/@react-aria/slider/docs/useSlider.mdx

```diff
@@ -0,0 +1,280 @@
+  height: 3px;
+  top: 13px;
+  width: 100%;
+}
+
+.filledRail {
+  position: absolute;
+  background-color: #707070;
+  height: 3px;
+  top: 13px;
+  inset-inline-start: 0;
+}
+
+.track {
+  position: relative;
+  height: 30px;
+  width: 100%;
+}
+
+.trackContainer {
+  position: relative;
+  width: 100%;
+}
+```
+
+
+```tsx example
+import {useSliderState} from '@react-stately/slider';
+import {FocusRing} from '@react-aria/focus';
+import {VisuallyHidden} from '@react-aria/visually-hidden';
+import {I18nProvider} from '@react-aria/i18n';
+
+function Slider(props) {
+  let trackRef = React.useRef(null);
+  let inputRef = React.useRef(null);
+  let origin = props.origin ?? props.minValue ?? 0;
+
+  let multiProps = {
+    ...props,
+    value: props.value == null ? undefined :  [props.value],
+    defaultValue: props.defaultValue == null ? undefined : [props.defaultValue],
+    onChange: props.onChange == null ? undefined : (vals) => props.onChange(vals[0]),
+    onChangeEnd: props.onChangeEnd == null ? undefined : (vals) => props.onChangeEnd(vals[0])
+  };
+
+  let state = useSliderState(multiProps);
+  let {
+    containerProps,
+    trackProps,
+    labelProps
+  } = useSlider(multiProps, state, trackRef);
+
+  let {thumbProps, inputProps} = useSliderThumb({
+    index: 0,
+    isDisabled: props.isDisabled,
+    trackRef,
+    inputRef
+  }, state);
+
+  let value = state.values[0];
+
+  return (
+    <I18nProvider locale="en-US">
+      <div className="slider" {...containerProps}>
+        <div className="sliderLabel">
+          {props.label && <label {...labelProps} className="label">{props.label}</label>}
+          <div className="value">{state.getThumbValueLabel(0)}</div>
+        </div>
+        <div className="trackContainer">
+          {
+            // We make rail, filledRail, and track siblings in the DOM, so that trackRef has no children.
+            // User must click on the trackRef to drag by track, and so it comes last in the DOM.
+          }
+          <div className="rail" />
+          <div
+            className="filledRail"
+            style={{
+              insetInlineStart: `${state.getValuePercent(Math.min(value, origin)) * 100}%`,
+              width: `${(state.getValuePercent(Math.max(value, origin)) - state.getValuePercent(Math.min(value, origin))) * 100}%`
+            }} />
+          <div ref={trackRef} className="track" {...trackProps} />
+          <FocusRing within focusRingClass="thumbFocusVisible" focusClass="thumbFocused">
+            <div
+              className="thumb"
+              style={{
+                insetInlineStart: `${state.getThumbPercent(0) * 100}%`
+              }}>
+              <div {...thumbProps} className="thumbHandle">
+                <VisuallyHidden><input className="input" ref={inputRef} {...inputProps} /></VisuallyHidden>
+              </div>
+            </div>
+          </FocusRing>
+        </div>
+      </div>
+    </I18nProvider>
+  );
+}
+
+<Slider
```

Comment:
The thumb doesn't seem to drag (at least on Chrome)

Reply:
?? it is for me